UI/GameSettingsScreen.cpp

```diff
@@ -106,6 +107,8 @@ bool IsBackendSupportHWTess() {
 	case GPU_BACKEND_VULKAN:
 	case GPU_BACKEND_DIRECT3D11:
 		return true;
+	case GPU_BACKEND_DIRECT3D9:
+		return DX9::dx_extensions.vertex_texture_fetch && DX9::dx_extensions.texture_float;
```

Comment:
We should find some more elegant way to check this (caps on the draw context perhaps) but I'm fine with an ifdef here too for now.